example/src/screens/Meeting/ActiveSpeakerView.tsx

```diff
@@ -75,11 +75,7 @@ const getActiveSpeakers = (
 ): PeerTrackNode[] => {
   const currentActiveSpeakers: PeerTrackNode[] = speakers.map(speaker => {
     const {peer} = speaker;
-    return {
-      id: peer.peerID + HMSTrackSource.REGULAR,
-      peer,
-      track: peer.videoTrack,
-    };
+    return createPeerTrackNode(peer);
```

Comment:
Active Speaker update will always come after onPeer & onTrack updates. 
When an active speaker update comes in a peerTrackNode will already exist. So new node should not be created, existing one should be used here